### ISSUE TITLE
Password is asked when running models tests

### ISSUE THREAD

**Issue by nubunto** [vattle/sqlboiler#47]
Hi there!

First of all, I want to start by thanking you guys for this great project.
Now, on to the helping part:

In my project, we're using gb as the primary build tool (although we experienced the same issues with the go tool)
Sqlboiler version is tag 2.0.1
```
$ gb test ./models
Password: *types password*
Password: *types password*
Password: Password: *types password*
(hangs forever)
```

This is my sqlboiler.toml:
```
[postgres]
dbname="mydbname"
host="localhost"
port=5432
user="postgres"
pass="postgres"
```

Any idea what could be happening?
Thanks!

**Comment by nullbio** [comment]
Hi @nubunto. Where is your sqlboiler.toml file located and where is your models folder located?

The test suite should not ask you for a password if one is defined in your sqlboiler.toml. Are you certain it is loading the correct sqlboiler.toml, not one without a password defined? You don't have a config defined anywhere else in your system, and you aren't using environment variables?

Also, what command did you use to generate your models?

**Comment by nubunto** [comment]
Hey!

So the tree for the project looks like this:
```
project/
|___ src/
     |____ sqlboiler.toml
     |____ models/
```

I tend to `cd` into src/ and run `sqlboiler -b schema_migrations postgres`

**Comment by nullbio** [comment]
Okay. That should be good. And you're certain there is a password defined in your sqlboiler.toml? As far as I'm aware, you should only ever get a password prompt if the password is missing from your config file.

**Comment by nubunto** [comment]
Yes, I'm 99% positive there is a "pass" field on sqlboiler.toml. Since sqlboiler generates the structs, I can assume it has a password, and it is valid. I will investigate a little more on this whenever I get the chance.

**Comment by aarondl** [comment]
We seem to have a recreate, will work on this. :)

**Comment by aarondl** [comment]
Found some bugs in the pgpassfile things. It was erroneously using my real pgpass file when I created and I didn't notice somehow. Sorry about that.

We also have to create a new entry in the pgpassfile for the "postgres" database, because of the way createdb/dropdb work, they attempt to switch to that database before performing any operations. We can't really get away from this since you can't drop a database without being on another database. Anyway it should be fixed in the dev branch @ 0dc68e749e0012553d44a16e661ea3cea1430368

**Comment by nullbio** [comment]
@nubunto We've released the fix in [2.0.2](https://github.com/vattle/sqlboiler/releases/tag/v2.0.2).

**Comment by nubunto** [comment]
Back with the tests, then :smile:
